lib/cdo/hamburger.rb

```diff
@@ -252,7 +251,6 @@ def self.get_header_contents(options)
       # When signed out, "Learn" will take all users to the student view of /courses.
       {title: I18n.t("#{loc_prefix}learn"), url: CDO.studio_url("/courses"), id: "header-learn"},
       {title: I18n.t("#{loc_prefix}teach"), url: CDO.code_org_url("/teach"), id: "header-teach"},
-      {title: I18n.t("#{loc_prefix}project_gallery"), url: CDO.studio_url("/projects/public"), id: "header-projects"},
```

Comment:
Will this be re-added once we have a page with featured projects? We'd still want signed out users to be able to see those projects, yeah? I see a requirement in the product spec:

> The link https://studio.code.org/projects/public should show the featured project page for both signed in and signed out users.

Reply:
I was also wondering the same thing earlier and so asked @samantha-code specifically about this. She confirmed that even when the featured gallery will be displayed instead of the public gallery, we want to remove the 'projects' tab from the page header for signed out users. This was a request from @dmcavoy and is related to what @breville discussed above (limited space).
Signed out users will still be able to have access to 'projects/public' by clicking on the 'Create' button at the top right corner and then clicking on 'View all projects...'.

Reply:
Thanks for clarifying!

Reply:
Can we hold on this till later today so I can talk to Sam?

Reply:
As next steps I think it would be easiest for @kelbyhawn to make all the changes to the header so that we don't remove this header item and then replace it a couple days later. So @fisher-alice could you remove that change here?

Reply:
Thanks @dmcavoy and @fisher-alice! Can the test code [here](https://github.com/code-dot-org/code-dot-org/pull/56666/files#diff-0a3070df50dbd552edc92085dd6733f35acfb7c982d39fcb3c9aab71bc0531a1) also be put back? I'm updating this in [my PR](https://github.com/code-dot-org/code-dot-org/pull/56527/files#diff-0a3070df50dbd552edc92085dd6733f35acfb7c982d39fcb3c9aab71bc0531a1).